ui/pages/settings/experimental-tab/experimental-tab.component.js

```diff
@@ -190,9 +192,49 @@ export default class ExperimentalTab extends PureComponent {
     );
   }
 
+  renderTransactionSecurityCheckToggle() {
+    const { t } = this.context;
```

Comment:
You'll also need to add an entry to `ui/helpers/constants/settings.js` so that search works properly

Reply:
Fixed here: 9e2f82fe90e1f3c1d4bc0502ed901d191a0adefc